version/version.go

```diff
@@ -15,5 +15,5 @@
 package version
 
 var (
-	Version = "v0.6.0+git"
+	Version = "v0.7.0+git"
```

Comment:
@theishshah I'm confused why some of these 0.7.0 changes are showing up in this PR. I checked `master` to be sure and all of them are already there. Maybe some weirdness happened with merging or rebasing after 0.7.0 was released?

Reply:
Yeah you might want to squash merge onto master.